lib/Model.php

```diff
@@ -234,7 +235,7 @@ class Model
 	 * Constructs a model.
 	 *
 	 * When a user instantiates a new object (e.g.: it was not ActiveRecord that instantiated via a find)
-	 * then @var $attributes will be mapped according to the schema's defaults. Otherwise, the given
+	 * then <code>$attributes</code> be mapped according to the schema's defaults. Otherwise, the given
```

Comment:
if you're going to change this one, you might want to fix the similar references in the same sentence